spine/model/layer/cnn/ppn.py

```diff
@@ -484,9 +492,75 @@ def process_loss_config(self, mask_loss='CE', resolution=5.,
         else:
             raise ValueError(
                     f"Mask loss name not recognized: {mask_loss}")
+            
+            
+    @staticmethod
+    def get_ppn_positives(coords : torch.Tensor, 
+                          labels: torch.Tensor, 
+                          ppn_labels: torch.Tensor, 
+                          resolution: float,
+                          offset: int):
+        """Get ppn positive label mask, but with cluster restrictions.
+
+        Parameters
+        ----------
+        coords : torch.Tensor
+            N x 3 3D coordinates of the image voxels
+        labels : torch.Tensor
+            N x 1 tensor of the particle id label for each voxel
+        ppn_labels : torch.Tensor
+            N_p x 6 PPN label points 
+        resolution : float
+            Distance from a label point in pixels within which a voxel is
```

Comment:
Within which a voxel is what? Match docstring in the forward function.

Reply:
Done.